### PR TITLE
Added CSP related changes to ML Analytics

### PR DESCRIPTION
**Pull request by raghupathiguduri**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/ml-analytics-service/tasks/main.yml [hunk 1]
```diff
@@ -105,6 +105,10 @@
     dest: "{{ config_path }}/config.ini"
     backup: yes
 
+- name: Copy GCP Secrets to JSON file
+  copy: dest="{{config_path}}/{{ml_GCP_secret_json_file}}" content="{{ ml_GCP_Secrets | to_nice_json}}" mode=0644 owner="{{ USER }}"
```
Comment (santhosh-tg) [inline ansible/roles/ml-analytics-service/tasks/main.yml:109]: @raghupathiguduri can we change the mode to **mode=0400** only the owner will need the read permission,right ?

Comment (raghupathiguduri) [reply ansible/roles/ml-analytics-service/tasks/main.yml]: Changed the mode @santhosh-tg

File: ansible/roles/ml-analytics-service/tasks/main.yml [hunk 2]
```diff
@@ -105,6 +105,10 @@
     dest: "{{ config_path }}/config.ini"
     backup: yes
 
+- name: Copy GCP Secrets to JSON file
+  copy: dest="{{config_path}}/{{ml_GCP_secret_json_file}}" content="{{ ml_GCP_Secrets | to_nice_json}}" mode=0400 owner="{{ USER }}"
```
Comment (santhosh-tg) [inline ansible/roles/ml-analytics-service/tasks/main.yml:109]: @raghupathiguduri  where are these variables ml_GCP_secret_json_file, ml_GCP_Secrets defined ?

File: ansible/roles/ml-analytics-service/tasks/main.yml [hunk 3]
```diff
@@ -105,6 +105,10 @@
...
+  when: ML_Cloud_Service_Provider == 'GCP'
```
Comment (santhosh-tg) [inline ansible/roles/ml-analytics-service/tasks/main.yml:110]: @raghupathiguduri  shouldn't the value be **gcloud** ?

**Review by santhosh-tg — APPROVED** [review]

**Pull request merged**